### DISCUSSION TITLE
Canva

### DISCUSSION THREAD

**Discussion by vikrant0705** [community/community#38348] [GitHub Education]
I could not finding Canva in student developers pack

**Comment by vikrant0705** [comment]
Where is Canva

**Comment by Joyin2** [comment]
> I could not finding Canva in student developers pack

Please search properly, I got canva

**Comment by SilasRodrigues19** [comment]
@vikrant0705 Some time ago Canva disappeared from the list of benefits of the GitHub Student Developer Pack, I believe it is no longer one of the benefits. We can see in [this link](https://education.github.com/pack), it no longer appears.

**Comment by samus-aran** [comment]
Hi @vikrant0705  👋 ! Thanks for the great question about Global Campus Benefits.

We no longer provide the Canva benefit as part of the Student Developer Pack. You can check out all the Student Developer Pack's latest offers and partners available [here](https://education.github.com/pack?sort=az#offers). For students who have already claimed for their 12 month benefit, you will continue to have access for the remainder of your 12 month offer period and can contact [Canva's support team](https://www.canva.com/help/get-in-touch/contact-support/) with any related questions.
